src/components/pages/PendingApplications/PendingApplication.css

```diff
@@ -10,12 +10,43 @@
   padding-top: 20%;
 }
 .note-button-color {
```

Comment:
This seems to be the same as line 45... why have 2 different ones? Maybe consider putting them into the same place